binaryaudit/marinerABIdiff.py

```diff
@@ -0,0 +1,19 @@
+import RPM
+import run
+
+import sys
+
+def getRPMList(dirPath):
+    ret = run.runCommand(dirPath, "ls")
+    print "Return code: ", ret.returncode
```

Comment:
Is this a debug output overseen? In the normal `sys.exit()` should be good enough to forward an error to the caller.

Thanks